app/models/event.rb

```diff
@@ -45,4 +49,40 @@ def end_after_start
       errors.add(:enddate, "must be after startdate.")
     end
   end
+
+  # Everything below this is leagues only code and will be moved to Leagues.rb once there is an actual option to create Leagues AND Tourneys, etc.
+
+  def add_test_teams
+    self.max_teams.times do |index|
+      self.teams << Team.new(name: "Team #{index}", private: false)
```

Comment:
`self.` is not actually neccessary here and you do not use anywhere else, so please maintain constistent code style.